src/components/paragraphs/stanford-banner/banner-paragraph.tsx

```diff
@@ -1,11 +1,14 @@
-import React, {HtmlHTMLAttributes} from "react";
+import React, {HtmlHTMLAttributes, ElementType} from "react";
 import {ParagraphStanfordBanner} from "@lib/gql/__generated__/drupal.d";
 import {H2, H3, H4} from "@components/elements/headers";
 import Wysiwyg from "@components/elements/wysiwyg";
-import Button from "@components/elements/button";
 import {getParagraphBehaviors} from "@components/paragraphs/get-paragraph-behaviors";
 import {twMerge} from "tailwind-merge";
-import HeroBanner from "@components/patterns/hero-banner";
+import clsx from "clsx";
```

Comment:
Looks like prettier isn't doing what it's supposed to. make sure to run `yarn install` if you merge/update to 1.x branch.